owlplug-client/src/main/java/com/owlplug/store/controllers/StoreController.java

```diff
@@ -344,14 +344,25 @@ public boolean installBundle(ProductBundle bundle) {
         bundle.getProduct().getName(), 
         bundle.getName());
 
+    
     String baseDirectoryPath = storeService.getBundleInstallFolder(bundle);
     String relativeDirectoryPath  = this.getPreferences().get(ApplicationDefaults.STORE_DIRECTORY_KEY, "");
 
+    Boolean shouldGroupByCreator = this.getPreferences().getBoolean(ApplicationDefaults.STORE_BY_MAKER_ENABLED_KEY, false);
+
+    //if the enduser wishes to group plugins by their creator,
+    //then we need to include the subdirectory as well.
+    if(shouldGroupByCreator) {
+        String creator = bundle.getProduct().getCreator();
+        relativeDirectoryPath = relativeDirectoryPath + File.separator + creator;
```

Comment:
Maybe we should sanitize the `creator` variable here to remove forbidden chars for a directory name. Probably linked with issue #56 😉 
You can use the function [FileUtils.sanitizeFileName](https://github.com/DropSnorz/OwlPlug/blob/2e43c1ff4f7bd9406c1ff744dcda748cc267f57e/owlplug-client/src/main/java/com/owlplug/core/utils/FileUtils.java#L56)

Reply:
that worked, I also noticed it pulled the spaces out of the folder names as well, but I think that's just fine